src/common/components/giftViews/giftUpdateView/giftUpdateView.tpl.html

```diff
@@ -96,51 +111,65 @@
 
     <div ng-if="$ctrl.gift.frequency === 'Quarterly'">
       <p>
-        You have selected <strong>quarterly</strong> as a frequency. Please tell
-        us in which months you would like your quarterly gifts to be made
+        <span translate>You have selected </span>
+        <strong translate>quarterly </strong>
+        <span translate>as a frequency. Please tell
+        us in which months you would like your quarterly gifts to be made</span>
       </p>
 
-      <div class="form-group">
-        <label class="radio">
+      <fieldset class="form-group">
+        <legend class="sr-only">Select quarterly gift months</legend>
+        <label class="radio" for="quarterlyMonth1">
           <input
+            id="quarterlyMonth1"
             type="radio"
+            name="quarterlyStartMonth"
             ng-model="$ctrl.gift.startMonth"
             ng-value="$ctrl.startDate($ctrl.gift.transactionDay, $ctrl.gift.nextDrawDate).format('M')"
           />
           {{ $ctrl.quarterlyMonths($ctrl.gift.transactionDay,
           $ctrl.gift.nextDrawDate) }}
         </label>
-        <label class="radio">
+        <label class="radio" for="quarterlyMonth2">
           <input
+            id="quarterlyMonth2"
             type="radio"
+            name="quarterlyStartMonth"
             ng-model="$ctrl.gift.startMonth"
             ng-value="$ctrl.startDate($ctrl.gift.transactionDay, $ctrl.gift.nextDrawDate, 1).format('M')"
           />
           {{ $ctrl.quarterlyMonths($ctrl.gift.transactionDay,
           $ctrl.gift.nextDrawDate, 1) }}
         </label>
-        <label class="radio">
+        <label class="radio" for="quarterlyMonth3">
           <input
+            id="quarterlyMonth3"
             type="radio"
+            name="quarterlyStartMonth"
             ng-model="$ctrl.gift.startMonth"
             ng-value="$ctrl.startDate($ctrl.gift.transactionDay, $ctrl.gift.nextDrawDate, 2).format('M')"
           />
           {{ $ctrl.quarterlyMonths($ctrl.gift.transactionDay,
           $ctrl.gift.nextDrawDate, 2) }}
         </label>
-      </div>
+      </fieldset>
     </div>
 
     <div ng-if="$ctrl.gift.frequency === 'Annual'">
       <p>
-        You have selected <strong>annually</strong> as a frequency. Please tell
-        us in which months you would like your annual gifts to be made
+        <span translate>You have selected </span>
+        <strong translate>annually </strong>
+        <span translate>as a frequency. Please tell
+        us in which months you would like your annual gifts to be made.</span>
       </p>
 
       <div class="row">
         <div class="col-md-6">
           <div class="form-group">
+            <label for="annualStartMonth" class="sr-only" translate>Select annual gift month</label>
```

Comment:
prettier can format HTML weirdly and put the closing `>` on a new line. Do you mind fixing that here and some other places in this file?